.gitignore

```diff
@@ -16,3 +16,6 @@ build
 .vscode
 output/
 .env
+*.svg
```

Comment:
I had these originally and replaced with `output/` but OK I guess it makes sense to keep them too :-)